tech/languages/python/python-installation.md

```diff
@@ -32,17 +32,17 @@ print('Hello World!')
 
 If you want to **exit** Python, press `Ctrl` + `D`.
 
-To run a program written in Python type `python3`, the path and the name of the program. Like this:
+To run a program written in Python, type `python3` followed by the path and name of the program. Like this:
 
 ```bash
 $ python3 example.py
 ```
 
 ## Using virtualenv
 
-When you work at some project it is good to keep it inside a virtual environment. It will keep the dependencies you need at one place and you no dot have to worry about different projects which needs different version of the same module.
+When you work with a project, it is good to keep it inside a virtual environment. It will keep the dependencies you need in one place and you do not have to worry about different projects which need different versions of the same module.
```

Comment:
Shouldn't this be 'on' a project?
(I know it implies you're actually working on it, but isn't that the case?)
`with` does not fit well there with me.

Reply:
'with' and 'on' have slightly different implications, but either works in this context.  I am fine with changing this to 'on'

Reply:
Sure :) Thanks.